src/cgnstypes.h.in

```diff
@@ -46,7 +46,7 @@
 # define CG_LONG_T @CGLONGT@
 #endif
 
-#ifdef BUILD_HDF5
+#if CG_BUILD_HDF5
```

Comment:
This breaks enabling parallel in CGNS, BUILD_HDF5 is what is used in CGNS proper. CG_BUILD_HDF5 is used only for configure.

Reply:
Hmm. IIRC, I was seeing issues where `BUILD_HDF5` was not consistently set and available, possibly from usage of CGNS. I'll look into it later this week.

Reply:
I've reverted it for now so that our daily testing passes. My understanding is CG_BUILD_* are only used in configure.bat (which is not tested and probably does not work).

Reply:
OK, I'll try reverting it in ParaView's import and see what changes.